### PR TITLE
#131 - partnerships page

### LINKED ISSUE #131: Add partnerships section
With joining [SoDA](https://sodapl.com/) and co-organizing Laravel Poland Meetup, it would be good to make a page or section somewhere to show off.

### PR DESCRIPTION
**Pull request by kamilpiech97**
This should close #131 .
Added partnerships page.
**Preview:**
obraz

### CODE REVIEW

File: src/locale/en.json [hunk 1]
```diff
@@ -129,6 +130,24 @@
...
       }
     },
+    "partnerships": {
+      "soda": {
+        "header": "Software Development Association Poland",
+        "lead": "Software Development Association Poland (SoDA) is an organization aimed at developing cooperation between software companies. It is the oldest and the largest organization of IT Service Employers in Poland, which is a mine of contacts and knowledge helping to develop the activities of companies operating in our industry. Thus, we are among the hundred most important companies on the market with a profile similar to ours."
+      },
+      "pwsz": {
+        "header": "Collegium Witelona Uczelnia Państwowa",
+        "lead": "As part of the cooperation, we strive to cooperate in the improvement of study programs in the fields of Information Technology and Production and Logistics Engineering. In addition, we declare the possibility of carrying out practical classes (internships, projects), a constant willingness to accept students of selected fields of study for apprenticeships and the possibility of cooperation in the development and implementation of educational projects."
```
Comment (EwelinaLasowy) [inline src/locale/en.json:140]:
```suggestion
        "lead": "As part of the cooperation, we strive to cooperate in the improvement of study programs in the fields of Information Technology and Production and Logistics Engineering. In addition, we declare the possibility of carrying out practical classes (internships, projects), a constant willingness to accept students of selected fields of study for apprenticeships, and the possibility of cooperation in the development and implementation of educational projects."
```

File: src/locale/en.json [hunk 2]
```diff
@@ -129,6 +130,24 @@
...
+      },
+      "laravel-meetup": {
+        "header": "Laravel Poland Meetup",
+        "lead": "An active community of programmers from all over Poland who want to expand their knowledge about Laravel and PHP. Together, we create meetups, which are held regularly every two months, we present interesting topics and fund prizes."
+      },
+      "phpers": {
+        "header": "PHPers",
+        "lead": "A community that is fascinated by, works with, or uses the PHP programming language as a hobby. Together, we create meetups that are held regularly in Wrocław and Poznań, we co-create interesting presentations and sponsor meetings."
```
Comment (EwelinaLasowy) [inline src/locale/en.json:148]:
```suggestion
        "lead": "A community that is fascinated by works with, or uses the PHP programming language as a hobby. Together, we create meetups that are held regularly in Wrocław and Poznań, we co-create interesting presentations and sponsor meetings."
```
```suggestion
        "lead": "A community that is fascinated by, works with, or uses the PHP programming language as a hobby. Together, we create meetups that are held regularly in Wrocław and Poznań, we co-create interesting presentations and sponsor meetings."
```

File: src/locale/en.json [hunk 3]
```diff
@@ -129,6 +130,24 @@
         "lead": "We fully understand that the software has to solve business problems: optimize tasks and resources, secure data, enable further scaling, and make life easier. Nowadays, most or practically all processes taking place in companies are often based on IT systems. We want to help make sure everything works as well as possible."
       }
     },
+    "partnerships": {
+      "soda": {
+        "header": "Software Development Association Poland",
+        "lead": "Software Development Association Poland (SoDA) is an organization aimed at developing cooperation between software companies. It is the oldest and the largest organization of IT Service Employers in Poland, which is a mine of contacts and knowledge helping to develop the activities of companies operating in our industry. Thus, we are among the hundred most important companies on the market with a profile similar to ours."
+      },
+      "pwsz": {
```
Comment (EwelinaLasowy) [inline src/locale/en.json:138]: Maybe we should call it "cwup" or "witelonka"?

File: src/pages/partnerships.svelte [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<Meta title={ $_('title.services') } description={ $_('description.services') }/>
```
Comment (EwelinaLasowy) [inline src/pages/partnerships.svelte:1]: I think a title should be "Partnerships".

File: src/locale/en.json (hunk 3 above)
Comment (kamilpiech97) [reply src/locale/en.json:138]: changed to `cwup`

File: src/pages/partnerships.svelte (hunk 4 above)
Comment (kamilpiech97) [reply src/pages/partnerships.svelte]: changed.

File: src/locale/en.json [hunk 5]
```diff
@@ -22,13 +23,15 @@
     "company": "Blumilk spółka z ograniczoną odpowiedzialnością based in Legnica, NIP: 6912551135, KRS: 0000866755.",
     "contact": "Do you have an idea for a project? Contact us via the form or visit us in the heart of Legnica!",
     "privacy": "Find out how we process your data.",
-    "services": "We specialize in creating web applications to support your business."
+    "services": "We specialize in creating web applications to support your business.",
+    "partnerships": "We cooperate with with SoDA, Collegium Witelona State University  and Laravel Poland Meetup, creating together an interesting and experience-rich meeting."
```
Comment (EwelinaLasowy) [inline src/locale/en.json:27]:
```suggestion
    "partnerships": "We cooperate with with SoDA, Collegium Witelona State University and Laravel Poland Meetup, creating together an interesting and experience-rich meeting."
```

File: src/locale/pl.json [hunk 6]
```diff
@@ -22,13 +23,15 @@
     "company": "Blumilk spółka z ograniczoną odpowiedzialnością z siedzibą w Legnicy, NIP: 6912551135, KRS: 0000866755.",
     "contact": "Masz pomysł na projekt? Skontaktuj się z nami poprzez formularz lub odwiedź nas w samym sercu Legnicy!",
     "privacy": "Dowiedz się, w jaki sposób przetwarzamy Twoje dane osobowe.",
-    "services": "Specjalizujemy się w tworzeniu aplikacji internetowych, które mają wspomóc Twój biznes."
+    "services": "Specjalizujemy się w tworzeniu aplikacji internetowych, które mają wspomóc Twój biznes.",
+    "partnerships": "Współpracujemy m.in. z SoDA, Collegium Witelona Uczelnia Panśtwowa Laravel Poland Meetup tworząc wspólnie ciekawe i bogate w doświadcenie spotkania."
```
Comment (EwelinaLasowy) [inline src/locale/pl.json:27]:
```suggestion
    "partnerships": "Współpracujemy m.in. z SoDA, Collegium Witelona Uczelnia Państwowa oraz Laravel Poland Meetup tworząc wspólnie ciekawe i bogate w doświadczenie spotkania."
```

File: src/locale/en.json (hunk 5 above)
Comment (krzysztofrewak) [inline src/locale/en.json]:
```suggestion
    "partnerships": "We cooperate with with SoDA, Collegium Witelona State University and Laravel Poland Meetup, creating together an interesting and experience-rich meeting."
```

Comment (krzysztofrewak) [reply src/locale/en.json]: Official name: http://www.bw.pwsz.legnica.edu.pl/artykul-154-about_university-0

File: src/locale/en.json [hunk 7]
```diff
@@ -129,6 +132,24 @@
...
+      },
+      "cwup": {
+        "header": "Collegium Witelona Uczelnia Państwowa",
+        "lead": "As part of the cooperation, we strive to cooperate in the improvement of study programs in the fields of Information Technology and Production and Logistics Engineering. In addition, we declare the possibility of carrying out practical classes (internships, projects), a constant willingness to accept students of selected fields of study for apprenticeships, and the possibility of cooperation in the development and implementation of educational projects."
+      },
+      "laravel-meetup": {
+        "header": "Laravel Poland Meetup",
+        "lead": "An active community of programmers from all over Poland who want to expand their knowledge about Laravel and PHP. Together, we create meetups, which are held regularly every two months, we present interesting topics and fund prizes."
+      },
+      "phpers": {
```
Comment (krzysztofrewak) [inline src/locale/en.json:148]: I would remove it form here now. We can add it in the future when we'd sponsor more than one meetup.

File: src/locale/en.json [hunk 8]
```diff
@@ -129,6 +132,24 @@
...
       }
     },
+    "partnerships": {
+      "soda": {
+        "header": "Software Development Association Poland",
+        "lead": "Software Development Association Poland (SoDA) is an organization aimed at developing cooperation between software companies. It is the oldest and the largest organization of IT Service Employers in Poland, which is a mine of contacts and knowledge helping to develop the activities of companies operating in our industry. Thus, we are among the hundred most important companies on the market with a profile similar to ours."
+      },
+      "cwup": {
+        "header": "Collegium Witelona Uczelnia Państwowa",
+        "lead": "As part of the cooperation, we strive to cooperate in the improvement of study programs in the fields of Information Technology and Production and Logistics Engineering. In addition, we declare the possibility of carrying out practical classes (internships, projects), a constant willingness to accept students of selected fields of study for apprenticeships, and the possibility of cooperation in the development and implementation of educational projects."
```
Comment (krzysztofrewak) [inline src/locale/en.json:142]: "As part of the cooperation, we strive to cooperate..."? The entire section looks like it was written in Polish and Google-translated into English. @jsawo @EwelinaLasowy maybe you could help here?

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]
Please crop and scale partnerships images to the same size.

File: src/locale/en.json (hunk 8 above)
Comment (EwelinaLasowy) [reply src/locale/en.json:142]: Maybe something like: "As part of collaboration, we strive to cooperate..."?

File: src/locale/en.json (hunk 7 above)
Comment (kamilpiech97) [reply src/locale/en.json]: okay, done it.

File: src/locale/en.json (hunk 8 above)
Comment (kamilpiech97) [reply src/locale/en.json]: Changed on proposition of @EwelinaLasowy

**Comment by kamilpiech97** [comment]
> Please crop and scale partnerships images to the same size.

My skill of croping and scaling images is on low level ;c but I scaled them by CSS. I need images with high resoultions because if I will show 300x300 px logo image on FHD screen that will look awful - I think.

**Comment by krzysztofrewak** [comment]
obraz

I refactored a little your changes to fit more into our website architecture.

**Comment by kamilpiech97** [comment]
> obraz
> 
> I refactored a little your changes to fit more into our website architecture.

Where did you find that logo of Collegium Witelona? :thinking:  On site they have low quality logo (jpg).

**Comment by kamilpiech97** [comment]
Where did you find that logo of Collegium Witelona? thinking On site they have low quality logo (jpg).

**Comment by krzysztofrewak** [comment]
> Where did you find that logo of Collegium Witelona? thinking On site they have low quality logo (jpg).

http://www.pwsz.legnica.edu.pl/strona-150-logo-1

**Review by krzysztofrewak — APPROVED** [review]

File: src/locale/en.json [hunk 9]
```diff
@@ -129,6 +132,20 @@
         "lead": "We fully understand that the software has to solve business problems: optimize tasks and resources, secure data, enable further scaling, and make life easier. Nowadays, most or practically all processes taking place in companies are often based on IT systems. We want to help make sure everything works as well as possible."
       }
     },
+    "partnerships": {
+      "soda": {
+        "header": "Software Development Association Poland",
+        "lead": "Software Development Association Poland (SoDA) is an organization aimed at developing cooperation between software companies. It is the oldest and the largest organization of IT Service Employers in Poland, which is a mine of contacts and knowledge helping to develop the activities of companies operating in our industry. Thus, we are among the hundred most important companies on the market with a profile similar to ours."
+      },
+      "cwup": {
+        "header": "Collegium Witelona Uczelnia Państwowa",
```
Comment (EwelinaLasowy) [inline src/locale/en.json:141]:
```suggestion
        "header": "Witelon Collegium State University",
```

**Review by krzysztofrewak — APPROVED** [review]

**Review by EwelinaLasowy — APPROVED** [review]

**Pull request merged**